app/views/user_mailer/invitation.html.erb

```diff
@@ -5,7 +5,7 @@
 
   <p>
     <% if !@user_name.blank? %>
-      <%= t ".has_invited", user_name: style_bold(@user_name) %>
+      <%= t(".has_invited", user_name: style_bold(@user_name).html_safe) %>
```

Comment:
Sorry, this still won't work -- the final `)` needs to be after `style_bold(@user_name)` and before `.html_safe`: `<%= t(".has_invited", user_name: style_bold(@user_name)).html_safe %>`